### PR TITLE
add missing sass call in gulp build task

### PR DESCRIPTION
**Pull request by andidev**

### CODE REVIEW

File: generators/client/templates/gulpfile.js [hunk 1]
```diff
@@ -291,7 +291,7 @@ gulp.task('serve', function () {
 });
 
 gulp.task('build', ['clean'], function (cb) {
-    runSequence(['copy', 'wiredep:app', 'ngconstant:prod'<% if(enableTranslation) { %>, 'languages'<% } %>], 'inject', 'assets:prod', cb);
+    runSequence(['copy', 'wiredep:app', 'ngconstant:prod'<% if(useSass) { %>, 'sass'<% } %><% if(enableTranslation) { %>, 'languages'<% } %>], 'inject', 'assets:prod', cb);
```
Comment (deepu105) [inline generators/client/templates/gulpfile.js]: It wasnt missing its already called by assets:prod task as dependency and having here would make it run twice thats why i didn't put it here

**Comment by deepu105** [comment]
Im reverting this

File: generators/client/templates/gulpfile.js (hunk 1 above)
Comment (deepu105) [inline generators/client/templates/gulpfile.js]: Btw assets has styles as dependency which has sass as dependency so that its run in order

**Comment by andidev** [comment]
Sorry that's right. My gulp skills lack

**Pull request merged**